content/reference/endpoints/fees.md

```diff
@@ -24,30 +29,33 @@ original_currency      | String  | Read       | Fee's original currency, list of
-rate_kind              | String  | Read/Write |  ** Required** Fee's rate kind, list of fee's rate kinds described in [enums section](/reference/enums#fee-rate-kinds).
-maximum_possible_rate  | [Decimal](/reference/enums#formats) | Read/Write | Fee's maximum possible rate, calculated value won't exceed amount present on this attribute. It is applied only for percentage kinds, list of kinds is described in [enums section](/reference/enums#fee-kinds)
------------------------|---------|------------|------------
-archived_at            | [Time](/reference/enums#formats) | Read         | Fee's archive time.
-created_at             | [Time](/reference/enums#formats) | Read         | Fee's create time.
-updated_at             | [Time](/reference/enums#formats) | Read         | Fee's update time.
+Name                   | Type    | Read/Write | Description | Constraints
+-----------------------|---------|------------|-------------|
+account                | Integer | Read       | Account's ID related to the Fee |
+-----------------------|---------|------------|------------|
+id                     | Integer | Read       | Fee's id. |
+currency               | String  | Read/Write | Requested fee's currency, defaults to `original_currency`. List of currencies codes described in [enums section](/reference/enums#currencies) | Must be one of the currencies enum 
+downpayment_percentage | [Decimal](/reference/enums#formats) | Read/Write |  Fee's downpayment percentage. | **Required**, must be great than or equal to: 0, must be less than or equal to: 100
+exchange_rate          | [Decimal](/reference/enums#formats) | Read       | The rate used to convert from the `original_currency` to `currency`. |
+name                   | [Object](/reference/enums#formats)   | Read/Write | Fee's name, list of locales described in [enums section](/reference/enums#locales). | **Required**, cannot contain non-printing characters 
+kind                   | String  | Read/Write | Fee's kind, list of fee's kinds described in [enums section](/reference/enums#fee-kinds) | **Required** must be one of the fee kinds
+original_currency      | String  | Read       | Fee's original currency, list of currencies codes described in [enums section](/reference/enums#currencies) |
+rate                   | [Decimal](/reference/enums#formats)  | Read/Write | Fee's rate. | **Required**, greater than or equal to: 0 
+rate_kind              | String  | Read/Write |  Fee's rate kind, list of fee's rate kinds described in [enums section](/reference/enums#fee-rate-kinds). | **Required**, must be one of the fee rate kinds 
+maximum_possible_rate  | [Decimal](/reference/enums#formats) | Read/Write | Fee's maximum possible rate, calculated value won't exceed amount present on this attribute. | Must be greater than or equal to: 0, applicable only for It is applied only for percentage kinds (list of kinds is described in [enums section](/reference/enums#fee-kinds)).
```

Comment:
`applicable only for It is applied only for percentage kinds`
looks like some duplication